setup.py

```diff
@@ -9,6 +10,21 @@
     changes = f.read()
 
 
+install_requires = [
+    'pytest',
+    'pep8',
+    'pyflakes',
+    'coverage',
+    'sphinx',
+    'pytest-cov',
+]
+
+if version_info.major == 2:
+    install_requires.append([
+        'mock',
+        'future',
```

Comment:
I don't think I actually ended up using any of the packages that `future` provides (I was originally using it to provide a consistently named `builtins` module, but now doing a conditional import instead), so I'll probably remove this. Just putting this note here so I don't forget :)